rpm/packetfence.spec

```diff
@@ -286,7 +286,7 @@ Requires: perl(Test::NoWarnings), perl(Test::ParallelSubtest)
 # required for the fake CoA server
 Requires: perl(Net::UDP)
 # For managing the number of connections per device
-Requires: haproxy >= 1.8.9, keepalived >= 1.4.3
+Requires: haproxy >= 1.8.9, keepalived >= 2.0.0
```

Comment:
The package doesn't seem to be in the devel repo yet

Reply:
Yes it's on the OBS , will push it on devel repo

Reply:
Done http://inverse.ca/downloads/PacketFence/CentOS7/devel/x86_64/RPMS/keepalived-2.0.20-1.1.x86_64.rpm